src/views/shared/SecurityBeacon.js

```diff
@@ -120,7 +140,8 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
         <div class="circle right"></div>\
       </div>\
     </div>\
-    <div class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
+    <div aria-live="polite" role="alert" class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
```

Comment:
The codepen demo works for me. I think there are just too many things going on at once in the actual loginpage. I've tried many, many variants with slightly different arrangements of DOM elements, text() vs html() vs adding/removing a node from the DOM. None of them work for me in this case without `role=alert`.